<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BookingProcess</class>
 <widget class="QStackedWidget" name="BookingProcess">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>518</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StackedWidget</string>
  </property>
  <property name="currentIndex">
   <number>2</number>
  </property>
  <widget class="QWidget" name="bookingPage">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="BookingView" name="bookingForm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>RÃ©servation</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="spc">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="resNextBtn">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../../../res/ressources.qrc">
          <normaloff>:/icons/icons/chevron-right.svg</normaloff>:/icons/icons/chevron-right.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="clientPage">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="ClientView" name="clientForm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Client</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <spacer name="spc_3">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="cliNextBtn">
        <property name="text">
         <string>Next</string>
        </property>
        <property name="icon">
         <iconset resource="../../../res/ressources.qrc">
          <normaloff>:/icons/icons/chevrons-right.svg</normaloff>:/icons/icons/chevrons-right.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="vehiclePage">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="VehicleListView" name="vehicleForm">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Vehicule</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="spc_2">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="saveBtn">
        <property name="text">
         <string>Enregistrer</string>
        </property>
        <property name="icon">
         <iconset resource="../../../res/ressources.qrc">
          <normaloff>:/icons/icons/save.svg</normaloff>:/icons/icons/save.svg</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VehicleListView</class>
   <extends>QGroupBox</extends>
   <header>src/views/VehicleList/VehicleListView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookingView</class>
   <extends>QGroupBox</extends>
   <header>src/views/Booking/BookingView.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ClientView</class>
   <extends>QGroupBox</extends>
   <header>src/views/Client/ClientView.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../res/ressources.qrc"/>
 </resources>
 <connections/>
</ui>
